x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/formatted_field.tsx

```diff
@@ -130,9 +132,12 @@ const FormattedFieldValueComponent: React.FC<{
         className={classNames}
         fieldName={fieldName}
         value={value}
-        tooltipProps={{ position: 'bottom', className: dataGridToolTipOffset }}
+        tooltipProps={
+          isUnifiedDataTable ? undefined : { position: 'bottom', className: dataGridToolTipOffset }
```

Comment:
This one here is for this I believe. In the current timeline table we have this tooltip, essentially showing the date in multiple formats. This would be a deprecation with the new table, but not sure how many people use it

<img width="390" alt="Screenshot 2024-03-08 at 1 10 26 PM" src="https://github.com/elastic/kibana/assets/17211684/9d8a045c-bc53-4633-aa4b-30a3ddd16f90">

Reply:
It is not a deprecation is now available in the new table as well.

Reply:
![Uploading grafik.png…]()